### PR TITLE
update Tutorials/Making_a_measurement/ example_codes

### PR DESCRIPTION
**Pull request by sansanda**
Pull request for upating the code on Tutorials:Making a measurement to be compatible with the new version of the Keithley 2400. Related issue --> #728

### CODE REVIEW

**Comment by bmoneke** [comment]
Thank you for adjusting the example.

One question though: Why do you change the number of samples and other settings?

**Comment by sansanda** [comment]
> Thank you for adjusting the example.
> 
> One question though: Why do you change the number of samples and other settings?

@bmoneke Hi. Just for increasing the speed of the tests on our Keithley.

Thanks

File: docs/tutorial/procedure.rst [hunk 1]
```diff
@@ -31,35 +32,43 @@ These steps are expressed in code as follows. ::
...
-    max_current = 0.01
+    averages = 10
+    max_current = 0.001
     min_current = -max_current
 
+    # Set source_current and measure_voltage parameters
+    current_range = 10e-3  # in Amps
+    compliance_voltage = 10  # in Volts
+    measure_nplc = 0.1  # Number of power line cycles
+    voltage_range = 1  # in VOlts
```
Comment (bmoneke) [inline docs/tutorial/procedure.rst:43]:
```suggestion
    voltage_range = 1  # in Volts
```

File: docs/tutorial/procedure.rst [hunk 2]
```diff
@@ -308,49 +317,52 @@ Now that you have a background on how to use the different features of the Proce
...
+        averages = IntegerParameter('Averages', default=8)
+        max_current = FloatParameter('Maximum Current', units='A', default=0.001)
+        min_current = FloatParameter('Minimum Current', units='A', default=-0.001)
 
         DATA_COLUMNS = ['Current (A)', 'Voltage (V)', 'Voltage Std (V)']
 
         def startup(self):
             log.info("Connecting and configuring the instrument")
-            self.sourcemeter = Keithley2400("GPIB::4")
+            self.sourcemeter = Keithley2400("GPIB::24", includeSCPI=True)
```
Comment (bmoneke) [inline docs/tutorial/procedure.rst:338]:
```suggestion
            self.sourcemeter = Keithley2400("GPIB::24")
```
Either an instrument supports SCPI or it does not. Therefore it should be called only in an instrument implementation, not when you create an instrument.

**Review by bmoneke — CHANGES_REQUESTED** [review]
Two small changes.

Did I infer correctly, that you ran these examples and they work?

File: docs/tutorial/procedure.rst (hunk 2 above)
Comment (sansanda) [reply docs/tutorial/procedure.rst:338]: Ok. Got. Thanks

**Comment by sansanda** [comment]
> Two small changes.
> 
> Did I infer correctly, that you ran these examples and they work?

Yes. Correct. They work.

**Comment by bmoneke** [comment]
Thanks for your contribution.

**Pull request merged**